src/frontends/onnx/README.md

```diff
@@ -0,0 +1,75 @@
+# OpenVINO ONNX Frontend
+
+The main responsibility of ONNX Frontend is import of ONNX models and conversion of these into the `ov::Model` representation. 
+Other capabilities of the ONNX Frontend:
+* modification of tensors properties (like data type and shapes)
+* changing topology of the models (like cutting subgraphs, inserting additional inputs and outputs)
+* user-friendly searching the models (via tensors and operators names)
+
+The component is written in `C++`, `Python` bindings are also available.
+If you want to contribute, please follow [the common coding style rules](../../../docs/dev/coding_style.md).
+
+
+## Key contacts
+
+In case of any questions, review and merge requests, please contact [openvino-onnx-frontend-maintainers](https://github.com/orgs/openvinotoolkit/teams/openvino-onnx-frontend-maintainers) group.
+
+
+## Components
+
+ONNX Frontend implements an interface common for all frontends defined in [Frontends API](./src/frontends/common/include/openvino/frontend).
+For backward compatibility reasons, the ONNX importer API (more lower-level abstraction approach) is still maintained. It can be found in [ONNX Importer](.src/frontends/onnx/frontend/include/onnx_import/onnx.hpp).
+
+The crucial place in ONNX Frontend is [the directory](.src/frontends/onnx/frontend/src/op) where the operators are implemented. Each operator handler has to be registered in [ops bridge](.src/frontends/onnx/frontend/src/ops_bridge.cpp). More details about it you can find in [How to add an new operation](./docs/how_to_add_op.md). Expect that ONNX frontend has capabilities to register a custom op by a user. It can be achieved via [ConversionExtension](.include/openvino/frontend/onnx/extension/conversion.hpp).
```

Comment:
```suggestion
The crucial place in ONNX Frontend is [the directory](./src/frontends/onnx/frontend/src/op) where the operators are implemented. Each operator handler has to be registered in [ops bridge](./src/frontends/onnx/frontend/src/ops_bridge.cpp). More details about it you can find in [How to add an new operation](./docs/how_to_add_op.md). Expect that ONNX frontend has capabilities to register a custom op by a user. It can be achieved via [ConversionExtension](./include/openvino/frontend/onnx/extension/conversion.hpp).
```